src/main/scala/com/databricks/spark/xml/XmlInputFormat.scala

```diff
@@ -97,25 +92,18 @@ private[xml] class XmlRecordReader extends RecordReader[LongWritable, Text] {
     val codec = new CompressionCodecFactory(conf).getCodec(path)
     if (null != codec) {
       decompressor = CodecPool.getDecompressor(codec)
-      // Use reflection to get the splittable compression codec and stream. This is necessary
```

Comment:
Not a big change. I just removed the reflection part that was needed for Hadoop 1.x.